templates/node/package.json.twig

```diff
@@ -42,12 +42,14 @@
   },
   "devDependencies": {
     "@types/node": "20.11.25",
+    "@types/json-bigint": "^1.0.4",
     "tsup": "7.2.0",
     "esbuild-plugin-file-path-extensions": "^2.0.0",
     "tslib": "2.6.2",
     "typescript": "5.4.2"
   },
   "dependencies": {
+    "json-bigint": "^1.0.0",
```

Comment:
Do we need a depencdency? is it possible without?

Reply:
Yup. I don't think there's a built-in logic for managing big-ints.